include/clap/clap.h

```diff
@@ -28,44 +28,37 @@
 #include "entry.h"
 
 #include "factory/plugin-factory.h"
-#include "factory/draft/plugin-invalidation.h"
-#include "factory/draft/preset-discovery.h"
+#include "factory/preset-discovery.h"
 
 #include "plugin.h"
 #include "plugin-features.h"
 #include "host.h"
+#include "universal-plugin-id.h"
 
```

Comment:
These includes can be removed I think which removes some clutter.
Created a PR: #388 (checks passed)